src/components/Search/Search.tsx

```diff
@@ -44,6 +50,8 @@ export const Search = ({
     deprecationWarning('Search property small is deprecated.  Use size')
   }
 
+  const buttonText = i18n?.buttonText || 'Search'
```

Comment:
If you want, you can even declare a default for `i18n` (such as the `EN_US` used in `DatePicker`). Since this is only one field within `i18n`, I'm good with it as is.